packages/injective-testing/Cargo.toml

```diff
@@ -1,20 +1,20 @@
 [package]
-authors     = [ "Bartek Tofel <bartek@injectivelabs.org>" ]
+authors     = [ "Markus Waas <markus@injectivelabs.org>" ]
 description = "Testing utils for Injective Cosmwasm and core modules"
 edition     = "2021"
 license     = "Apache-2.0"
 name        = "injective-testing"
-repository  = "https://github.com/InjectiveLabs/cw-injective/tree/master/packages/injective-testing"
-version     = "0.1.5"
+repository  = "https://github.com/InjectiveLabs/cw-injective/tree/dev/packages/injective-testing"
+version     = "0.1.6"
 
 [dependencies]
-anyhow              = { version = "1.0.66" }
-base64              = { version = "0.13.1" }
-cosmwasm-std        = { version = "=1.2.7", features = [ "abort", "iterator", "stargate" ] }
-cw-multi-test       = { version = "0.16.2" }
-injective-cosmwasm = { version = "0.2.17" }
-injective-math     = { version = "0.2.3" }
-rand                = { version = "0.4.6" }
-secp256k1           = { version = "0.6.2" }
-serde               = { version = "1.0.137", default-features = false, features = ["derive"] }
-tiny-keccak         = { version = "1.2.1" }
+anyhow             = { version = "1.0.66" }
+base64             = { version = "0.13.1" }
+cosmwasm-std       = { version = "1.5.0", features = [ "abort", "cosmwasm_1_2", "cosmwasm_1_3", "cosmwasm_1_4", "iterator", "stargate" ] }
+cw-multi-test      = { version = "0.16.2" }
+injective-cosmwasm = { version = "0.2.18", path = "../injective-cosmwasm" }
+injective-math     = { version = "0.2.4", path = "../injective-math" }
```

Comment:
I realized it's more convenient via path. You definitely need the path temporarily when updating the full code base because packages are not published yet, at which point you may ask, why not always via path?